### PR TITLE
Added KiranAdhikari

### PR DESCRIPTION
**Pull request by ErKiran**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [ ] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by Nick-Vazquez** [comment]
Hey there, @ErKiran!

Just looking over your PR, and it looks like the picture you've added is not square. Would you mind either cropping your current image or adding a new one and updating the PR?

> ## 2. Add your profile picture to the folder

> Enter the folder you just created and add a profile picture of your choice. Accepted files are png and jpg, **should be squared and minimum size 544x544 pixels**.

Thanks for your submission!

**Comment by elisemoe** [comment]
Hi @ErKiran 👋😄 we can't wait to celebrate graduation with you! Be sure to submit the form before 12:00 AM PT today (in ~3hr) to be considered for swag.

**Comment by ErKiran** [comment]
> We were not able to find your form submission on our side. Fill a shipping form before we can move on with your revew.
Where can I find shipping form?

**Comment by ErKiran** [comment]
@elisemoe I have submitted the form. Please find it.

**Comment by ErKiran** [comment]
@Nick-Vazquez I have reuploaded the image. Please check it

File: _data/KiranAdhikari/KiranAdhikari.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Kiran Adhikari
+institution: Tribhuvan University
+profile_pic: KiranAdhikari.jpg
```
Comment (Mohammed-Shoaib) [inline _data/KiranAdhikari/KiranAdhikari.md:4]: Incorrect file name, please do one of the following:
1. Change `KiranAdhikari.jpg` to `kiranadhikari.png` in the markdown file.
2. Rename `kiranadhikari.png` to `KiranAdhikari.jpg`.

File: _data/KiranAdhikari/KiranAdhikari.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+quote: "Premature optimization is the root of all evil"
```
Comment (Mohammed-Shoaib) [inline _data/KiranAdhikari/KiranAdhikari.md:5]: Remove double quotes from begin and end.

Comment (ErKiran) [reply _data/KiranAdhikari/KiranAdhikari.md]: Done

File: _data/KiranAdhikari/KiranAdhikari.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+name: Kiran Adhikari
+institution: Tribhuvan University
+profile_pic: kiranadhikari.jpg
```
Comment (Mohammed-Shoaib) [inline _data/KiranAdhikari/KiranAdhikari.md:4]: Incorrect file name, please do one of the following:
1. Change `kiranadhikari.jpg` to `kiranadhikari.png` in the markdown file.
2. Rename `kiranadhikari.png` to `kiranadhikari.jpg`.
I'd suggest you make the change in the markdown file!

**Pull request merged**